### ISSUE TITLE
self. removed where necessary (v0.26.2)

### ISSUE THREAD

**Issue by charmaex** [nicklockwood/SwiftFormat#113]
I spotted a pattern where self. was falsely removed:

```Swift
class ViewController {

    private weak var mapView: Map!

    func viewDidLoad() {
        super.viewDidLoad()

        let camera = Camera()
        let mapView = Map()

        self.mapView = mapView

        if let cameraList = camera.all, cameraList.count > 0 {}
    }
}
```

In the upper code `self.mapView = mapView` turns to `mapView = mapView`.

**Comment by nicklockwood** [comment]
Thanks, I'll take a look.

**Comment by nicklockwood** [comment]
Fixed in 0.27.0
